test/unit/reflectivity_ui/interfaces/event_handlers/test_main_handler.py

```diff
@@ -52,7 +52,7 @@ def test_congruency_fail_report(self, data_server):
         # New files
         message = self.handler._congruency_fail_report([data_server.path_to('REF_M_38198.nxs.h5'),
                                                         data_server.path_to('REF_M_38199.nxs.h5')])
-        assert 'values for log S1HWidth that differ above tolerance 0.01' in message
+        assert 'values for log DANGLE that differ above tolerance 0.01' in message
```

Comment:
Did you manually change 'S1HWidth' to 'DANGLE'?

Reply:
The log name changed in the test